### PR TITLE
(fix) Fix broken styles in visits widget

### PR DESCRIPTION
**Pull request by CynthiaKamau**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Fix broken styles in visits widget

## Screenshots

https://www.loom.com/share/0cdc45d67a0943e296558edf53a7f5f2?sid=c2fb0955-8fac-413e-86bb-27a229c38457

### CODE REVIEW

File: packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@
   padding: 1rem 0rem;
 
   :global(.cds--tabs) {
-    max-height: 10rem;
+    min-height: 8rem;
```
Comment (donaldkibet) [inline packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.scss:26]:
```suggestion
    min-height: spacing.$spacing-08;
```

File: packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.scss [hunk 2]
```diff
@@ -50,7 +50,7 @@
   outline-offset: 0;
 
   &:active, &:focus {
-    outline: 2px solid var(--brand-03) !important;
+    outline: 3px solid var(--brand-03) !important;
```
Comment (donaldkibet) [inline packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.scss:53]:
```suggestion
    outline: spacing.$spacing-03 solid var(--brand-03) !important;
```

File: packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.scss [hunk 3]
```diff
@@ -62,7 +62,7 @@
 
   &[aria-selected="false"] {
     border-bottom: none;
-    border-left: 2px solid $ui-03;
+    border-left: 3px solid $ui-03;
```
Comment (donaldkibet) [inline packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.scss:65]:
```suggestion
    border-left: spacing.$spacing-03 solid $ui-03;
```

**Review by donaldkibet — CHANGES_REQUESTED** [review]

File: packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.scss (hunk 2 above)
Comment (CynthiaKamau) [reply packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.scss:53]: `spacing.$spacing-03` is 8px

**Review by donaldkibet — APPROVED** [review]

**Pull request merged**